js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+userInfoSettings=Benutzer Einstellungen
+connectionAndAuthentication=Verbindung & Authentifizierung
+encryption=Verschlüsselung
+enableStartTLS=StartTLS aktivieren
+testConnectionHint.withoutEmail=Um die Verbindung zu testen, müssen Sie zunächst eine E-Mail-Adresse für den aktuellen Benutzer ({{userName}}) konfigurieren.
+testConnectionHint.withoutEmailAction=E-Mail Adresse konfigurieren
+testConnection=Verbindung testen
+smtpPortPlaceholder=SMTP Port (Standardwert 25)
+envelopeFromHelp=Eine E-Mail-Adresse, die für Bounces (nicht zustellbaren E-Mails) verwendet wird (optional).
+resetPasswordAllowed=Passwort vergessen
+disabled=Nicht aktiv
+themes=Thema
+selectATheme=Thema auswählen
+loginTheme=Thema für Anmeldung
+loginThemeHelp=Wählen Sie ein Thema für die Anmeldeseiten, Einmalpasswörter (OTP), Zustimmungen, Registrierung und Passwort vergessen.
+accountTheme=Thema für Benutzerkonto
+accountThemeHelp=Wählen Sie ein Thema für die Benutzerkonto-Verwaltungskonsole.
+adminTheme=Thema für Administration
+adminThemeHelp=Wählen Sie ein Thema für die Administration.
+emailTheme=Thema für E-Mails
+emailThemeHelp=Wählen Sie ein Thema für E-Mails, die vom Server gesendet werden.
+keys=Schlüssel
+keysList=Schlüsselliste
+providers=Anbieter hinzufügen
+keysFilter.ACTIVE=Aktive Schlüssel
+keysFilter.PASSIVE=Passive Schlüssel
+keysFilter.DISABLED=Deaktivierte Schlüssel
+algorithm=Algorithmus
+use=Verwendung
+provider=Anbieter
+validTo=Gültig bis
+publicKeys=Öffentlicher Schlüssel
+eventListeners=Ereignis-Listener
+eventListenersHelpTextHelp=Ereignis-Listener die Ereignisse für den Realm behandeln.
+userEventsSettings=Benutzerereignisse
+adminEventsSettings=Administrationsereignisse
+saveEvents=Ereignisse speichern
+save-user-eventsHelp=Wenn diese Option aktiviert ist, werden die Benutzerereignisse in der Datenbank gespeichert, wodurch die Ereignisse für die Administrations- und Benutzerkontokonsolen verfügbar werden.
+revert=Zurücksetzen
+clearUserEvents=Benutzerereignisse löschen
+user-clearEventsHelp=Löscht alle Benutzerereignisse in der Datenbank.
+refresh=Aktualisieren
+addSavedTypes=Gespeicherte Ergeignis-Typen hinzufügen
+searchEventType=Gespeicherte Ereignisart suchen
+eventType=Art des gespeicherten Ereignisses
+expiration=Ablaufdatum
+expirationHelp=Legt das Ablaufdatum für Ereignisse fest. Abgelaufene Ereignisse werden in regelmäßigen Abständen aus der Datenbank gelöscht.
+on=An
+off=Aus
+clearAdminEvents=Admin-Ereignisse löschen
+admin-clearEventsHelp=Löscht alle Verwaltungsereignisse in der Datenbank.
+includeRepresentation=Repräsentation speichern
+includeRepresentationHelp=JSON-Darstellung für Erstellungs- und Aktualisierungsanfragen speichern.
+internationalization=Internationalisierung
+internationalizationHelp=Wenn diese Option aktiviert ist, können Sie wählen, welche Gebietsschemata Sie für diesen Bereich unterstützen und welches Gebietsschema standardmäßig verwendet werden soll.
```

Comment:
Wouldn't Gebietsschema be the wrong term here? I'd say Sprachen would be a better fit.

Reply:
I looked this up from the microsoft website and they used Gebietschema to translate a locale.
However I agree that "Sprache" would be more appriopriate here.